auth-web/src/views/auth/UserProfileView.vue

```diff
@@ -65,7 +62,7 @@ export default class UserProfileView extends Mixins(NextPageMixin) {
 
   private navigateBack (): void {
     if (this.currentOrganization) {
-      this.$router.push(`/account/${this.currentOrganization.id}`)
+      window.location.replace(ConfigHelper.getBcrosDashboardURL())
```

Comment:
Any idea the difference between this and `window.location.assign(url)` ? Is one preferable over the other?

Reply:
Good question: https://stackoverflow.com/questions/4505798/difference-between-window-location-assign-and-window-location-replace#:~:text=The%20difference%20is%20how%20history,history%2C%20%22assign%22%20will.&text=According%20to%20MDN%3A,button%20to%20navigate%20to%20it.

Lets change to assign to preserve the session history

Reply:
Nice, I hadn't thought about that before.

This is the back link. Does assign make sense? I dunno!

Reply:
(For forward links, yes, preserving history makes sense.)

Reply:
OK I can't decide, so it's up to you!

Reply:
Yea, i think assign makes more sense. I don't see a reason to not preserve the history, especially since we are one big happy (app) family.